server-x/probes/liveness.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash -e
+curl -sSfo /dev/null --max-time 10 "http://localhost:8080/auth/health/live"
```

Comment:
The Quarkus distribution is placing Keycloak directly at the root of the URL, so no `/auth` context is needed here. The same for readiness check.

Reply:
Ok. I'll fix that. Updating [the spec](https://github.com/keycloak/keycloak-community/pull/55/files) would be a good idea.

Reply:
Thanks, I've added a comment to the spec PR. Approving.